great-idea/product.html

```diff
@@ -0,0 +1,138 @@
+<h1>Product</h1>
+<br>
+<p>Our services provide the best digital value in nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque eleifend. Mauris euismod facilisis iaculis.</p>
+</div>
+</div>
+<br>
+
+
+
+<div class="menu-container">
+
+<div class = "item" >
+    
+<h2>Digital Design</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+
+<div class="item">
+  <h2>UX / UI</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+
+<div class="item">
+<h2>Digital Marketing</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+<div class="item">
+    <h2>Web Development</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+<div class="item">
+    <h2>iOS Development</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+
+<div class="item">
+<h2>Android Development</h2>
+<br>
+<p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+eleifend. Mauris euismod facilisis iaculis.</p>
+<br>
+<button class = "learn">Learn More</button>
+</div>
+<div class = "line"></div>

```

Comment:
You could also use a [horizontal rule](https://devdocs.io/html/element/hr) here instead of a div, which provides a bit more semantic meaning.